unity-renderer/Assets/DCLPlugins/RealmsPlugin/RealmPlugin.cs

```diff
@@ -41,6 +43,7 @@ private void RealmChanged(AboutConfiguration current, AboutConfiguration _)
 
```

Comment:
Let's create an extension method to call `current.IsWorld()`, that will hide this variable